llvm/test/CodeGen/SystemZ/builtin-setjmp-longjmp-alloca-02.ll

```diff
@@ -0,0 +1,323 @@
+; Test for Frame Pointer in first slot in jmp_buf.
```

Comment:
Similar question here.  This is a very long test, and only a couple of instructions are tested via CHECK statements.  Whatever we want to test here should be done in a much more streamlined test case.